source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3516,6 +3534,12 @@ export abstract class Input extends CardElement implements IInput {
                 }
             }
 
+            this.updateVisualState(true /*initialRender*/);
```

Comment:
can you use named parameter for maintainers?

Reply:
fixed